packages/app/src/pages/dashboard/migrate.tsx

```diff
@@ -0,0 +1,314 @@
+			onClick: handleUnstakeKwenta,
+			active: claimableBalance.lte(0) && stakedKwentaBalance.gt(0),
+		},
+		{
+			key: 'step-3',
+			copy: t('dashboard.stake.tabs.migrate.step-3-copy'),
+			label: t('dashboard.stake.tabs.migrate.staked'),
+			value: truncateNumbers(stakedKwentaBalanceV2, 2),
+			buttonLabel: kwentaStakingV2Approved ? t('dashboard.stake.tabs.migrate.stake') : 'Approve',
+			onClick: handleStakeKwenta,
+			active: claimableBalance.lte(0) && stakedKwentaBalance.lte(0),
+		},
+	]
+
+	const MIGRATE_CARDS: StakingCards[] = [
+		{
+			category: t('dashboard.stake.portfolio.balance.title'),
+			card: [
+				{
+					key: 'balance-liquid',
+					title: t('dashboard.stake.portfolio.balance.liquid'),
+					value: truncateNumbers(kwentaBalance, 2),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+				{
+					key: 'balance-staked',
+					title: t('dashboard.stake.portfolio.balance.staked'),
+					value: truncateNumbers(stakedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.rewards.title'),
+			card: [
+				{
+					key: 'rewards-claimable',
+					title: t('dashboard.stake.portfolio.rewards.staking'),
+					value: truncateNumbers(claimableBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+				{
+					key: 'rewards-trading',
+					title: t('dashboard.stake.portfolio.rewards.trading'),
+					value: truncateNumbers(kwentaRewards, 4),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v2'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestableV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v1'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalance, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestable, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+	]
+
+	return (
+		<>
+			<Head>
+				<title>{t('dashboard-stake.page-title')}</title>
+			</Head>
+			<StakingHeading>
+				<FlexDivCol rowGap="5px">
+					<StyledHeading variant="h4">Migrate to Staking V2</StyledHeading>
+					<Body color="secondary">
+						Lorem ipsum dolor sit amet consectetur. Ut in nisl ut quam condimentum lacus.
+					</Body>
+				</FlexDivCol>
+				<StyledButton
```

Comment:
Can we have disabled / loading states on button

Reply:
This is a link to the docs so there is no need to disable it.
![image](https://github.com/Kwenta/kwenta/assets/4819006/41295a0e-5ede-4411-90d4-ce40fd98a201)